index.html

```diff
@@ -1670,7 +1671,19 @@ <h6 id="async-action-response">Asynchronous Action Response</h6>
               "timeRequested": "2021-11-10T11:43:19.135Z"
             }
           </pre>
-        </section>
+          <div class="rfc2119-assertion" id="http-basic-profile-protocol-binding-invokeaction-22">
+          <p>
+            In resource constrained environments, if the buffer becomes full then a Web Thing SHOULD
+            remove the oldest Action result ActionStatus from its buffer to make room for a newer one.
```

Comment:
On reflection the term "buffer" implies certain implementation details about the data structure which may not apply to all implementations. I would suggest a more generic term like "list", since in the API the collection of action requests is represented as a list/array of `ActionStatus` objects. There are many ways this could be modelled in different programming languages on the back end.

Also, it should be made clear that only completed/failed actions should be removed to make space. How to prioritise older vs. newer pending or running action requests needs to be application specific.

Also, not every `ActionStatus` object will be the same size, so it may be necessary to remove more than one old object to make room for one new object.

```suggestion
            In resource constrained environments, the ActionStatus objects of older completed/failed actions MAY be deleted to make room for newly invoked actions.
```

To be honest, I think what remains is already covered by the [Note](https://w3c.github.io/wot-profile/#h-note-3) at the end of the section.